setup.py

```diff
@@ -5,12 +5,12 @@
 setup(
     name='ebi_eva_common_pyutils',
     packages=find_packages(),
-    version='0.2.8',
+    version='0.3.2',
```

Comment:
Just a heads up that you might have to rebase with the appropriate version after [Andres' PR](https://github.com/EBIvariation/eva-common-pyutils/pull/6) is merged...

Reply:
Yes and it is something we should try to agree on going forward